# Fazal2020Scripts
Scripts used to process and aggregate ExpansionHunter Denovo output for a large number of samples. These scripts were used to create Supplementary Tables 1 and 2 in Fazal, et al (currently in submission) and are included here for reproducibility. 
This repository also includes bed files designed to be loaded as custom annotations in IGV or the UCSC genome browser to visualize the repetitive loci analyzed in the paper. The IGV tracks include information on repeat motif, allele frequency, mean, median, standard deviation, and median absolute deviation of anchored in-repeat-reads observed among the samples for each site. 
